framework/composer.json

```diff
@@ -77,6 +77,9 @@
         "bower-asset/punycode": "1.3.*",
         "bower-asset/yii2-pjax": "~2.0.1"
     },
+    "suggest": {
```

Comment:
I think we can require it safely. It's light dependency containing interface only.

Reply:
There is no need for this package to be mandatory.
In case particular project does not use PSR middleware - this package is a simple garbage, which is never used. In case some project does use some PSR middleware - it is more likely to coming from another composer package, which will have 'http-server-middleware' among his dependencies for sure.

Reply:
We're potentially converting action filters to middlewares, aren't we?

Reply:
I insist on having it in `require`.

Reply:
> We're potentially converting action filters to middlewares, aren't we?

At the present state, we are not. If such changeset will appear, it will include composer.json changeset inside it.

> I insist on having it in require.

Unless the official PSR-15 composer package appear this is risk prone. I can agree having mandatory dependency on `psr/http-server-middleware` but not on `http-interop/http-server-middleware`

Reply:
OK. Makes sense. I hope PHP-FIG will finish PSR-15 soon and there will be official interface package available.